From 3f0e57a048402e60f783daa0bc4c2e2af5804df6 Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Thu, 2 Mar 2017 15:52:17 +0800
Subject: [PATCH 1573/5242] MLK-14407 arm: dts: enable lpi2c7 & disable lpi2c4

commit  c91e0430f6275a55e274e98084eef1fc82877bdf from
https://source.codeaurora.org/external/imx/linux-imx.git

lpi2c4 is not used on imx7ulp platform, disable it;
lpi2c7 is used for touch, enable it;

Signed-off-by: Gao Pan <pandy.gao@nxp.com>
(cherry-pick from 9137ed6de38513c585206febe7ce6c8279674f1b)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    1 +
 arch/arm/boot/dts/imx7ulp.dtsi    |    1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index 149c2ba..fb71dcc 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -369,6 +369,7 @@
 	#size-cells = <0>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_lpi2c7>;
+	status = "okay";
 
 	focaltech@38 {
 		compatible = "focaltech,fts";
diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index 0a537c7..d76040e 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -223,6 +223,7 @@
 			assigned-clocks = <&clks IMX7ULP_CLK_LPI2C4>;
 			assigned-clock-parents = <&clks IMX7ULP_CLK_FIRC>;
 			assigned-clock-rates = <48000000>;
+			status = "disabled";
 		};
 
 		lpi2c5: lpi2c5@402C0000 {
-- 
1.7.9.5

